styles.css

```diff
@@ -30,8 +34,8 @@
     .logo{
```

Comment:
Prueba con esto, habiendo creado un contenedor para la imagen, es mejor cirtar asi a su css, simploficas el scrapping de las imagenes

```suggestion
    .logo-container>img{
```

Reply:
ok